<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addRecordDialog</class>
 <widget class="QDialog" name="addRecordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Выберите дату и время брони</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateEdit" name="dateEdit"/>
   </item>
   <item>
    <widget class="QTimeEdit" name="startTimeEdit"/>
   </item>
   <item>
    <widget class="QTimeEdit" name="endTimeEdit">
     <property name="dateTime">
      <datetime>
       <hour>23</hour>
       <minute>59</minute>
       <second>0</second>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="OKButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
